cmd/metal-api/main.go

```diff
@@ -732,11 +740,13 @@ func initRestServices(withauth bool) *restfulspec.Config {
 		logger.Fatal(err)
 	}
 
+	networkService := service.NewNetwork(logger.Named("network-service"), ds, ipamer, mdc)
```

Comment:
This line we can revert and put back directly into `restful.DefaultContainer.Add(networkService)`